src/webapp/utils/selectablePeriods.ts

```diff
@@ -0,0 +1,11 @@
+import _ from "lodash";
+import React from "react";
+
+export function useSelectablePeriods(startYear: number): string[] {
```

Comment:
it is a naming standard that only custom hooks begin with 'use'. Although this is not wrong, i recommend changing the function to getSelectablePeriods as it is util function.

Reply:
the `useSelectable` function is a custom hook. i have moved it to the webapp/hooks folder instead